versions/3.1.0.md

```diff
@@ -1453,14 +1455,14 @@ In contrast with the 2.0 specification, `file` input/output content in OpenAPI i
 # content transferred with base64 encoding
 schema:
   type: string
-  format: base64
+  contentEncoding: base64
 ```
 
 ```yaml
 # content transferred in binary (octet-stream):
 schema:
   type: string
-  format: binary
+  contentMediaType: application/octet-stream
```

Comment:
Should be include a `mediaTypeObject` in one of these examples? Is that the preferred OAS way to indicate media-type? What if there is a conflict between the `mediaTypeObject` key and a `contentMediaType` keyword?

Reply:
> What if there is a conflict between the `mediaTypeObject` key and a `contentMediaType` keyword?

So.... technically I would *not* use the schema in this example in the first place for a purely binary resource.  Because what this means is that there is a JSON string (double-quoted) with binary content, which doesn't actually make any sense.  You would have to encode the binary content into a legal JSON string (as indicated by `contentEncoding`).

For pure binary content, I would just us a Media Type Object without a schema.

Does that make sense?  I didn't want to start revamping larger things without first trying the minimal changes, as I assume this approach (`{"type": "string", "format": "binary"}`) has more or less worked for OAS so far.  But I wouldn't actually advise it, personally.

Reply:
@MikeRalphson @webron should I make the larger change here (to not use a Schema Object for binary content, and just use a Media Type Object)?